templates/minimal/runtime/Cargo.toml

```diff
@@ -13,7 +13,6 @@ publish = false
 codec = { workspace = true }
 scale-info = { workspace = true }
 polkadot-sdk = { workspace = true, features = [
-	"experimental",
```

Comment:
it works without this? 

Need to figure out the relationship between `polkadot-sdk` parent umbrella, and how it can be used to pull in `polkadot-sdk-frame` only. I thought `experimental` was needed here.

Reply:
`build` works, I don't know if there's anything else to check